xunit-autolabeler-v2/ast_parser/python/source_parsers/webapp2_router.py

```diff
@@ -0,0 +1,72 @@
+# Copyright 2020 Google LLC. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+
+from typing import Any, List
+
+
+from ast_parser.python.drift_data_dict import make_drift_data_dict
+
+
+def parse(nodes: List[Any]) -> List[Any]:
+    """Identify webapp2 route-handling snippets in Python
+       files and extract their language-agnostic data
+    Args:
+        nodes: A list of AST nodes obtained via the 'ast' package
+        class_name: the class name of the Python file being parsed
+    Returns:
+        A list of webapp2-handling methods (as AST nodes) with
+        added language-agnostic data (in the 'drift' attribute)
+    """
+
+    wsgi_configs = [x for x in nodes if
```

Comment:
Question:

In other 2 files, you're deduping.
In this file, you don't care? Why?

Reply:
See above - I'm happy to add a dupe check here though.

(There *shouldn't* be dupes here, so raising an exception if they exist - as `flask_router.py` does - would make sense to me.)

Reply:
Dupe check + relevant tests added.